### PR TITLE
Bug 1121416 - [Loop] Invitee name is not shown in history details when h...

### PR DESCRIPTION
**Pull request by crdlc**
...e joins as desktop user

### CODE REVIEW

File: app/js/screens/room/room_history.js [hunk 1]
```diff
@@ -168,7 +141,14 @@
           // We want desc sorting
           return b.date.getTime() - a.date.getTime();
         });
-        _elements.forEach(_renderEvt);
+        // The scroll based on chunks requires items have to be rendered
+        // once the previous one was done because of "_renderedIndex". Some
+        // items here take more time than others because of mozContacts so we
+        // have to wait for them.
```
Comment (crdlc) [inline app/js/screens/room/room_history.js]: You can see this problem when you have more than ten events and the first one is a "communication" event which uses mozContact when the participant has account.

File: app/js/screens/room/room_history.js [hunk 2]
```diff
@@ -168,7 +141,14 @@
...
+        var renderNextEvent = idx => {
+          _renderEvt(_elements[idx]).then(renderNextEvent.bind(null, ++idx));
+        };
```
Comment (mcjimenez) [inline app/js/screens/room/room_history.js]: NIT: the curly brackets are not needed here, please remove them

File: app/js/helpers/room/room_controller.js [hunk 3]
```diff
@@ -152,6 +152,9 @@
     _startCommunicationTime = new Date().getTime();
     _communicationEnd = false;
     _communicationToken = currentToken;
+    Rooms.get(currentToken).then(room => {
+      _participants = room.participants;
+    })
```
Comment (mcjimenez) [inline app/js/helpers/room/room_controller.js]: NIT: curly brackets are not needed here, please remove them

File: app/js/helpers/room/room_controller.js [hunk 4]
```diff
@@ -170,9 +173,9 @@
            i++) {
         // If participants[i] is not logged he hasn't account and we want
         // to keep RoomEvent identifier
-        if (_participants[i].account &&
-            _participants[i].account !== Controller.identity) {
-          other = _participants[i].account;
+        var account = _participants[i].account;
+        if (account && account !== Controller.identity) {
```
Comment (mcjimenez) [inline app/js/helpers/room/room_controller.js]: NIT: If (account && (account !== Controller.identity)) Although the parentheses are not needed I think It's clearer this way (I leave it to you if you want to changed it or not)

File: app/js/screens/room/room_history.js (hunk 2 above)
Comment (crdlc) [inline app/js/screens/room/room_history.js]: they are not needed, right, but it is clearer so if you don't have a strong inconvenient to remove them, I prefer to keep them here.

File: app/js/helpers/room/room_controller.js (hunk 3 above)
Comment (crdlc) [inline app/js/helpers/room/room_controller.js]: same

File: app/js/helpers/room/room_controller.js (hunk 4 above)
Comment (crdlc) [inline app/js/helpers/room/room_controller.js]: ok I add this, thx

**Pull request merged**